lib/Doctrine/Common/Annotations/AnnotationReader.php

```diff
@@ -307,7 +307,8 @@ public function getMethodAnnotation(ReflectionMethod $method, $annotationName)
      */
     private function getIgnoredAnnotationNames(ReflectionClass $class)
     {
-        if (isset($this->ignoredAnnotationNames[$name = $class->getName()])) {
+        $name = $class->getName();
```

Comment:
The expression was valid before - consider raising an issue in phpstan, since `$name` would have been defined in any case